### ISSUE TITLE
Segmentation fault on Apple M2

### ISSUE THREAD

**Issue by m-albert** [InsightSoftwareConsortium/ITKElastix#232]
After downgrading to itk-elastix==0.14.2 (pip install) as described [here](https://github.com/InsightSoftwareConsortium/ITKElastix/issues/136), installation worked on an Apple M2.

However, I get a segmentation fault (reproduced in fresh python 3.9 and 3.10 conda environments):

```
import itk
itk.ParameterObject.New()
```
> zsh: segmentation fault  python
/Users/malbert/miniconda3/envs/itk-elastix/lib/python3.9/multiprocessing/resource_tracker.py:216: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '

**Comment by thewtex** [comment]
Hi @m-albert , 0.17.3 wheels for macOS ARM were just uploaded. Please give them a try in a fresh environment and let us know how it goes for you. Thank you!

**Comment by m-albert** [comment]
Hi @thewtex, thank you very much!

I installed the new 0.17.3 macOS ARM wheel and the problem reported above disappeared. Also, performing registrations using both the procedural and object oriented APIs worked.

One unrelated problem I've noticed when going through the example notebooks is that after installing `itkwidgets` over pip (which installed itkwidgets-0.32.6), [this problem](https://github.com/InsightSoftwareConsortium/ITKElastix/issues/231) reappeared.
